torch/csrc/jit/script/parser.cpp

```diff
@@ -447,7 +447,7 @@ struct ParserImpl {
         auto range = L.next().range;
         auto expr = parseExp();
         L.expect(TK_NEWLINE);
-        return Raise::create(range, expr);
+        return Raise::create(range, Maybe<Expr>::create(range, expr));
```

Comment:
This is a bug fix that's exposed by adding this new functionality - `Raise::create` takes in a `Maybe<Expr>`, not an `Expr`. I'd expect the type system to catch this, but I guess there's some things that prevent it from doing so?

Reply:
`Maybe<T>` has an implicit constructor `Maybe(const T& tree)`, which I think is what was getting called before. What's the issue you were having here?

Reply:
Huh. Seems like it works. I guess I misattributed some other issue to this.